generator/testdata/rust/prost/golden/type/google.r#type.rs

```diff
@@ -0,0 +1,22 @@
+// This file is @generated by prost-build.
```

Comment:
Do we have any control over whether this file gets a license?

Reply:
I could not find any. I am not sure we are supposed to put a license in these files either.  We could add the license after the fact: #1421